packages/map-template/src/components/Search/Search.jsx

```diff
@@ -265,6 +283,32 @@ function Search({ onSetSize }) {
         }
     });
 
+    /*
+     * When useKeyboard parameter is present, add click event listener which determines when the keyboard should be shown or not. 
+     */
+    useEffect(() => {
+        if (useKeyboard) {
+            const onClick = (event) => {
+                // Detect the className of the keyboard buttons, the space and the backspace
+                // If the user clicks on the keyboard or the search fields, the keyboard should stay visible 
+                if (event.target.className === 'hg-button hg-standardBtn' ||
```

Comment:
This is very specific to the internals of the keyboard component, which may change in a future version.

Could we instead try something with the `closest()` method (https://developer.mozilla.org/en-US/docs/Web/API/Element/closest), for example `event.target.closest('mi-keyboard')` to catch the clicks on the keyboard?

Reply:
Very nice suggestion! I think it works well 👌🏻